Sources/Apollo/GraphQLGETTransformer.swift

```diff
@@ -42,7 +42,10 @@ public struct GraphQLGETTransformer {
       return nil
     }
 
-    components.queryItems = queryItems
+    if queryItems.count > 0 {
```

Comment:
note to myself to use `.apollo.isNotEmpty` here, but there are some shenanigans around the import I won't make you deal with

Reply:
I wasn't aware of `.apollo.isNotEmpty` until now. Have just updated the PR.

Reply:
Cool - I'll have to do some silliness to make cocoapods play nice later but i appreciate you doing this bit!